pillar.example

```diff
@@ -42,3 +45,11 @@ template:
     #     - 'example_alt.tmpl'
     #     - 'example_alt.tmpl.jinja'
 
+    # For testing purposes
+    source_files:
+      template-config-file-file-managed:
+        - 'example.tmpl.jinja'
+
+  # Just for testing purposes
+  winner: pillar
+  added_in_pillar: asdf
```

Comment:
Just to make it clear that is being pulled from the correct place, wouldn't it be better to have a different value than the one `added_from_lookup` ?  ie, `added_in_pillar: pillar_value`?

Reply:
@javierbertoli I'll update the PR. Thanks for the feedback!